src/core/backbone_router/bbr_manager.cpp

```diff
@@ -224,7 +224,6 @@ void Manager::HandleMulticastListenerRegistration(const Coap::Message &aMessage,
             uint32_t origTimeout = timeout;
 
             timeout = OT_MIN(timeout, static_cast<uint32_t>(Mle::kMlrTimeoutMax));
-            timeout = OT_MAX(timeout, static_cast<uint32_t>(Mle::kMlrTimeoutMin));
```

Comment:
But why is `kMlrTimeoutMax` appliable but `kMlrTimeoutMin` is not?
Seems not very consistent?

Reply:
To be honest I don't know where the `kMlrTimeoutMax` value comes from, but since you were the one adding it and have not opposition to remove it from here and in any case there seems to be no restriction from the spec in the Timeout TLV in MLR.req, I will remove the normalization completely.

Reply:
Thanks for the tip. It has been quite a while since the code was written so that I have to take a look at the code and spec again. 

The change of this PR looks good to me.

> To be honest I don't know where the `kMlrTimeoutMax` value comes from, but since you were the one adding it and have not opposition to remove it from here and in any case there seems to be no restriction from the spec in the Timeout TLV in MLR.req, I will remove the normalization completely.

I guess let's keep `kMlrTimeoutMax` as is for now. 

`kMlrTimeoutMax` was not from Thread Spec, but an implementation-specific limitation on MLR timeouts (about 24 days) to make Timer management simpler. 
So, we should keep `kMlrTimeoutMax` before we enhance the code.

Reply:
Fair enough.